tests/conftest.py

```diff
@@ -49,7 +49,7 @@ def _create_module_file(code, tmp_path, name):
 def disable_error_urls():
     # Don't add URLs during docs tests when printing
     # Otherwise we'll get version numbers in the URLs that will update frequently
-    os.environ['PYDANTIC_ERRORS_OMIT_URL'] = 'true'
+    os.environ['PYDANTIC_ERRORS_INCLUDE_URL'] = 'false'
```

Comment:
This could potentially be a breaking change (though I'm not sure who might have this enabled?)

Reply:
Could it, in a conftest? Based on the commit message for that bit in #9040 this also to silence a warning.